src/app/max-calc/max-calc.component.html

```diff
@@ -23,7 +30,7 @@
       Min is {{ minRealEstatePrice }}
     </mat-error>
   </mat-form-field>
-  <div class="slider-container">
+  <div *ngIf="helloSlider" class="slider-container">
```

Comment:
It seems to me that parts following each other with *ngIf="helloSlider" could be put into 1 big ng-container with *ngIf="helloSlider"

Reply:
I would've done that but there are divs that alternate between !helloSlider and helloSlider and the grid relies on their order. I'll try putting them in the same div and still have them retain their order in the grid when I have time